javatools_backend/src/main/java/org/xvm/notes.txt

```diff
@@ -0,0 +1,148 @@
+
+Ignore packages for now?
+
+
+* org/xvm/XEC.java - entry point
+
+* org/xvm/xtc - Read an XTC file support classes; ClassPart, *Part, CPool,Version,VerTree
+* org/xvm/xtc/cons - "cons" things from XTC files
+* org/xvm/xtc/AST  - "AST"  things from XTC files
+
+* org/xvm/xtc/ClzBuilder, ModBuilder, Javac, XType, XValue,
+
+* org/xvm/xec - generated class strings act "as if" loaded from here
+* org/xvm/xec - runtime support classes (no ties with gen classes)
```

Comment:
Should this file be merged into master? Should it be turned into GitHub issues or something instead?